force-app/main/default/classes/UserSelector.cls

```diff
@@ -35,10 +35,13 @@
 
 public with sharing class UserSelector {
 
-   public List<User> getUsers() {
+   public List<User> getActiveStandardUsers() {
       return [
           SELECT Id
-          FROM User LIMIT 10000
+          FROM User
+          WHERE IsActive = TRUE
+          AND UserType = 'Standard'
```

Comment:
Should this alert go to all `Standard` users or just those with some level of system admin permission, such as those with CustomizeApplication?

Reply:
The [WI ](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000RIKsYAO/view)calls for delivery to all Users, not just Admins, so I don't think the check for an Admin perm is necessary.